### DISCUSSION TITLE
Is it possible to integrate with an existing application?

### DISCUSSION THREAD

**Discussion by Char99s** [mynaparrot/plugNmeet-server#542] [General]
Hello everyone,

I have a learning management system, and I am looking to implement a live meeting feature. 
I successfully ran the plugNmeet server, and using the SDK I was able to create a room and generate a join token from my Nextjs app. 

My question is do I need to use the client application for the ui or for joining the meeings?

Can't I make the pages by myself in a specific route in my Next.js app?

**Comment by jibon57** [comment]
Hi,

You can follow: https://www.plugnmeet.org/docs/api/get-client-files

Thanks

**Reply by Char99s** [reply]
Will do, thank you!

**Reply by Char99s** [reply]
Any examples using JS? I have an allergy to php 😛

**Reply by jibon57** [reply]
PHP is just example, you can follow the doc to implement into any language.

**Reply by Char99s** [reply]
> PHP is just example, you can follow the doc to implement into any language.

I see, I got somewhere in my code but while trying it out i get null files for css and js. i get the status and msg though. any idea?

  const pnm = new PlugNmeet(
    "http://localhost:8080",
    "plugnmeet",
    "zumyyYWqv7KR2kUqvYdq4z4sXg7XTBD2ljT6"
  );
  const files = await pnm.getClientFiles();
  console.log("GOT FILES", files);
  
Logs below:

Received files 
{status: true, msg: 'success', css: null, js: null}
css: null
js: null
msg: "success"
status: true

Thanks!!

**Reply by jibon57** [reply]
You'll need to build client files by: `npm run build`

**Reply by Char99s** [reply]
> You'll need to build client files by: `npm run build`

Hello again, apologies for opening another issue I forgot I had this ongoing one. Thank you for helping out! I was able to get the client files after building the project as you said. I am now facing another problem using those files to integrate the UI. I replicated the code from conference.php to typescript but for some reason it just wouldn't work. Could it be because I am using typescript or does it have nothing to do?

I made sure that every step from the conference.php was followed correctly and I logged every value for the files and tags and scripts andd they all seem to be correct:

```
return (
    <>
      <Head>
        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
        <meta name="viewport" content="width=device-width,initial-scale=1" />
        <title>plugNmeet</title>
        {/* Preload CSS and JS */}
        {cssTagsPreload}
        {jsTagsPreload}
        {/* CSS and Configuration */}
        {cssTags}
        {cnfScript}
      </Head>
      <div>
        <div id="plugNmeet-app"></div>
        {/* JS Files */}
        {jsTags}
      </div>
    </>
  );
```
  
  here is how i'm returning them in the render function. Your help is greatly appreciated!
